src/api/agentd.js

```diff
@@ -45,4 +45,36 @@ export default (client: ApiRequester, baseUrl: string) => ({
       null,
       ApiRequester.successResponseParser,
     ),
+
+  staticLogin: () =>
+    client.post(
+      `${baseUrl}/users/me/agents/login`,
+      null,
+      null,
+      ApiRequester.successResponseParser,
+    ),
+
```

Comment:
Any reason to duplicate the `loginWithLineId` method?

Reply:
lol -- never realized `loginWithLineId` was already using the `/user/me/agents` route, i'd just blindly inserted placeholder code while waiting for the specs to formalize. Consider it fixed.